tools/README.md

```diff
@@ -77,6 +77,11 @@ A script that was never written.
 Given a directory full of `*.sql` table schemas, produces the SQL commands to
 replace all the foreign key references to `users.user_id` with `@buendia_admin`.
 
+#### `get_package_version.sh`
```

Comment:
A naming convention (that isn't completely followed! but) I would like to use is to put extensions on libraries, but no extensions on executables.  So `foo.py` is a Python module and `utils.sh` is a Bash function library, but `save_the_world` is a script you can run (and the implementation is free to switch languages).

Reply:
Fixed